### ISSUE TITLE
There is no name property for all the combo box inside Scroll Viewer(Preview) pane.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#186]
**Describe the bug**

**Pre-requisite: **
Turn on Narrator.
Set narrator to item mode.

**Steps to reproduce the bug**

1. Launch XAML Control Gallery app.
2. Navigate to Scroll Viewer button and activate it.
3. Navigate to any combo box and observe the Narrator reading.

**Expected behavior**

All combo box should have a Name property.

**Actual behavior**

When the user navigates to the combo box, Narrator reads only the name of the selected combo box item.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63896104-f80e8a00-c9a5-11e9-94f4-aec39a690792.png)

![image](https://user-images.githubusercontent.com/13142665/63896119-0066c500-c9a6-11e9-827d-3a0d8c78a7bf.png)

**Version Info**

Gallery version (found on Settings page): 

Version Info: 1.2.8.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/22205713/?triage=true)
